app/collections/create-form.cjsx

```diff
@@ -55,14 +67,14 @@ module?.exports = React.createClass
         <input className="collection-name-input" ref="name" onChange={@handleNameInputChange} placeholder="Collection Name" />
         <div className="collection-create-form-actions">
           <label>
-            <input ref="private" type="checkbox" defaultChecked={false}/>
+            <input ref="private" type="checkbox" onClick={@logPrivacy.bind this} defaultChecked={false}/>
```

Comment:
Binding here is throwing a React warning: `You are binding a component method to the component. React does this for you automatically in a high-performance way, so you can safely remove this call.` I don't think the binding here is necessary.